src/gui/MainWindow.cpp

```diff
@@ -826,6 +829,7 @@ void MainWindow::setMenuActionState(DatabaseWidget::Mode mode)
             m_ui->actionEntryAutoTypePassword->setEnabled(singleEntrySelected && dbWidget->currentEntryHasPassword());
             m_ui->actionEntryAutoTypePasswordEnter->setEnabled(singleEntrySelected
                                                                && dbWidget->currentEntryHasPassword());
+            m_ui->actionEntryAutoTypeTOTP->setEnabled(singleEntrySelected && dbWidget->currentEntryHasTotp());
```

Comment:
Should change this to "setVisible" since we don't even want to see the option if there is no TOTP defined for this entry.

Reply:
corrected, 
it had to bee enabled and visible as in 
```
            m_ui->actionGroupEmptyRecycleBin->setVisible(recycleBinSelected);
            m_ui->actionGroupEmptyRecycleBin->setEnabled(recycleBinSelected);
```